ui/article-renderer/src/main/java/org/cru/godtools/article/activity/ArticlesActivity.java

```diff
@@ -67,6 +73,10 @@ protected void onStart() {
     public void onArticleSelected(@Nullable final Article article) {
         Timber.tag("ArticlesActivity")
                 .d("Article selected: %s", article != null ? article.title : null);
+
+        assert article != null;
```

Comment:
asserting the article isn't null shouldn't be how you handle a null article being sent to this method.

The `assert mTool != null` works because the activity has a check state when it starts to make sure we have a `mTool` defined, otherwise the activity would never reach the state that this method would be called.